Fix test failures of awamicmp1 and awamicmp2.

Debian bug report:

https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=855814

Patch adapted from upstream source repository:

https://github.com/silnrsi/graphite/commit/f7bbdd87d510d587c872e314d6458160c0069c65

From f7bbdd87d510d587c872e314d6458160c0069c65 Mon Sep 17 00:00:00 2001
From: Martin Hosken <martin_hosken@sil.org>
Date: Tue, 14 Feb 2017 17:00:03 +0000
Subject: [PATCH] try -ffloat-store for fp stability. Remove nested const to
 work around VS bug

---
 src/CMakeLists.txt    | 2 +-
 src/inc/locale2lcid.h | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 4f1e7e5..5b61b5c 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -108,7 +108,7 @@ set_target_properties(graphite2 PROPERTIES  PUBLIC_HEADER "${GRAPHITE_HEADERS}"
 
 if  (${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
     set_target_properties(graphite2 PROPERTIES 
-        COMPILE_FLAGS   "-Wall -Wextra -Wno-unknown-pragmas -Wendif-labels -Wshadow -Wctor-dtor-privacy -Wnon-virtual-dtor -fno-rtti -fno-exceptions -fvisibility=hidden -fvisibility-inlines-hidden -fno-stack-protector"
+        COMPILE_FLAGS   "-Wall -Wextra -Wno-unknown-pragmas -Wendif-labels -Wshadow -Wctor-dtor-privacy -Wnon-virtual-dtor -fno-rtti -fno-exceptions -fvisibility=hidden -fvisibility-inlines-hidden -fno-stack-protector -ffloat-store"
         LINK_FLAGS      "-nodefaultlibs ${GRAPHITE_LINK_FLAGS}" 
         LINKER_LANGUAGE C)
     if (CMAKE_COMPILER_IS_GNUCXX)
